website/blog/2020-11/_index.md

```diff
@@ -0,0 +1,69 @@
+aliases: ["/blog/2020/11/04/00"]
+---
+
+Two months after our last Gardener release update, we are happy again to present release v1.11 and v1.12 in this blog post. Control plane migration, load balancer consolidation, new security features are just a few topics we progressed with. As always, a detailed list of features, improvements, and bug fixes can be found in the [release notes](https://github.com/gardener/gardener/releases) of each release. If you are going to update from a previous Gardener version, please take your time to go through the action items in the release notes.
+
+## Noteable Changes in v1.12
+
+Release v1.12, fresh from the oven, is shipped with plenty of improvements, features and some API changes we want to pick up in the next sections. 
+
+### Functionless DNS Providers ([gardener/gardener#3036](https://github.com/gardener/gardener/pull/3036))
+
+This release drops the support for so-called functionless DNS providers. Those are providers in a shoot’s specification (`.spec.dns.providers`) which don’t serve the shoot’s domain (`.spec.dns.domain`), but are created by Gardener in the seed cluster to serve DNS requests coming from the shoot cluster. If such providers don’t specify a `type` or `secretName` the creation or update request for the corresponding shoot is denied.
+
+### Seed Taints ([gardener/gardener#2955](https://github.com/gardener/gardener/pull/2955))
+
+With the introduction of `seed.spec.settings`, we spent the seed object a dedicated section in favor of previously used taints `disable-capacity-reservation, disable-dns, invisible`. These already deprecated taints were still considered and synced for a certain time which allowed operators to switch their integration to the new `settings` field. As of version v1.12, support for the mentioned taints has been dropped and are automatically removed from seed objects. You may use the actual taint names in a future release of Gardener again to serve your own use-cases.
+
+### Load Balancer Events During Shoot Reconciliation ([gardener/gardener#3028](https://github.com/gardener/gardener/pull/3028))
+
+As Gardener is capable of managing thousands of clusters, it is crucial to keep operation efforts at a minimum. This release demonstrates this endeavor by further improving error reporting to the end user. During a shoot’s reconciliation, Gardener creates `Services` of type `LoadBalancer` in the shoot cluster, e.g. for VPN or Nginx-Ingress addon, and waits for a successful creation. However, in the past we experienced that occurring issues from the party creating the load balancer (typically [Cloud-Controller-Manager]( https://kubernetes.io/docs/concepts/architecture/cloud-controller/)) are only exposed in the logs or as events. Gardener now fetches these event messages in propagates them to the shoot status in case of a failure. Users can then often fix the problem themselves, if for example the failure discloses an exhausted quota on the cloud provider.
```

Comment:
```suggestion
As Gardener is capable of managing thousands of clusters, it is crucial to keep operation efforts at a minimum. This release demonstrates this endeavor by further improving error reporting to the end user. During a shoot’s reconciliation, Gardener creates `Services` of type `LoadBalancer` in the shoot cluster, e.g. for VPN or Nginx-Ingress addon, and waits for a successful creation. However, in the past we experienced that issues caused by the party creating the load balancer (typically [Cloud-Controller-Manager]( https://kubernetes.io/docs/concepts/architecture/cloud-controller/)) are only exposed in the logs or as events. Gardener now fetches these event messages and propagates them to the shoot status in case of a failure. Users can then often fix the problem themselves, if for example the failure discloses an exhausted quota on the cloud provider.
```